gpii/node_modules/flowManager/src/BrowserChannel.js

```diff
@@ -34,7 +34,7 @@
 
                 gpii.request.flowManager.logAndNotify("Lifecycle manager returned: ", events.onError)(errMsg);
             } else {
-                gpii.settingsHandlers.webSockets.addClient(solutionId, fluidRequest.socket);
+                gpii.settingsHandlers.webSockets.addClient(solutionId, fluidRequest.socket, entries[solutionId].settingsHandlers[0].options);
```

Comment:
While you were gone, the big solution registry format pull request was merged (https://github.com/GPII/universal/commit/ab4ea0b4515606cd0999e8a860e7a48249ab8d30) which means that the settingshandlers entries are now keyed by an identifier, which can be referred to in the start, stop, configure, etc., blocks... so after merging with the latest master, this line breaks the tests